RHDEVDOCS-4472 -Add query to view metrics collected by Telemetry (#51059)
openshift/openshift-docs: modules/telemetry-showing-data-collected-from-the-cluster.adoc

[id="showing-data-collected-from-the-cluster_{context}"]
= Showing data collected by Telemetry

You can view the cluster and components time series data captured by Telemetry.

.Prerequisites

* You have installed the {product-title} CLI (`oc`).
* You have access to the cluster as a user with the `cluster-admin` role or the `cluster-monitoring-view` role.

.Procedure

> * Run the following query, which runs against a cluster's Prometheus service and returns the full set of time series data captured by Telemetry:

Reviewer: ```suggestion
* Run the following command, which returns the full set of time series data captured by Telemetry:
```

Author: fixed

--data-urlencode 'match[]={__name__="cluster:log_collected_bytes_total:sum"}' \
--data-urlencode 'match[]={__name__="cluster:log_logged_bytes_total:sum"}' \
--data-urlencode 'match[]={__name__="cluster:kata_monitor_running_shim_count:sum"}' \
--data-urlencode 'match[]={__name__="platform:hypershift_hostedclusters:max"}' \
--data-urlencode 'match[]={__name__="platform:hypershift_nodepools:max"}' \
--data-urlencode 'match[]={__name__="namespace:noobaa_unhealthy_bucket_claims:max"}' \
--data-urlencode 'match[]={__name__="namespace:noobaa_buckets_claims:max"}' \
--data-urlencode 'match[]={__name__="namespace:noobaa_unhealthy_namespace_resources:max"}' \
--data-urlencode 'match[]={__name__="namespace:noobaa_namespace_resources:max"}' \
--data-urlencode 'match[]={__name__="namespace:noobaa_unhealthy_namespace_buckets:max"}' \
--data-urlencode 'match[]={__name__="namespace:noobaa_namespace_buckets:max"}' \
--data-urlencode 'match[]={__name__="namespace:noobaa_accounts:max"}' \
--data-urlencode 'match[]={__name__="namespace:noobaa_usage:max"}' \
--data-urlencode 'match[]={__name__="namespace:noobaa_system_health_status:max"}' \

> --data-urlencode 'match[]={__name__="ocs_advanced_feature_usage"}'

Reviewer 2: missed `os_image_url_override:sum` according to:

https://github.com/openshift/cluster-monitoring-operator/blob/release-4.12/manifests/0000_50_cluster-monitoring-operator_04-config.yaml

since the new metrics would be added after the doc is merged, not sure if we have a better solution for the doc, otherwise we have to update the doc if new metrics  added later
